--- download-scannet.py	2020-05-05 10:08:35.608257161 +0100
+++ download-scannet_new.py	2020-05-05 10:09:52.701293293 +0100
@@ -4,8 +4,7 @@
 # -*- coding: utf-8 -*-
 import argparse
 import os
-#import urllib.request (for python3)
-import urllib
+import urllib.request
 import tempfile
 
 BASE_URL = 'http://kaldir.vc.in.tum.de/scannet/'
@@ -27,8 +26,7 @@
 
 
 def get_release_scans(release_file):
-    #scan_lines = urllib.request.urlopen(release_file)
-    scan_lines = urllib.urlopen(release_file)
+    scan_lines = urllib.request.urlopen(release_file)
     scans = []
     for scan_line in scan_lines:
         scan_id = scan_line.decode('utf8').rstrip('\n')
@@ -55,8 +53,7 @@
         fh, out_file_tmp = tempfile.mkstemp(dir=out_dir)
         f = os.fdopen(fh, 'w')
         f.close()
-        #urllib.request.urlretrieve(url, out_file_tmp)
-        urllib.urlretrieve(url, out_file_tmp)
+        urllib.request.urlretrieve(url, out_file_tmp)
         os.rename(out_file_tmp, out_file)
     else:
         print('WARNING: skipping download of existing file ' + out_file)
@@ -119,7 +116,7 @@
     print(TOS_URL)
     print('***')
     print('Press any key to continue, or CTRL-C to exit.')
-    key = raw_input('')
+    key = input('')
 
     if args.v1:
         global RELEASE
@@ -176,7 +173,7 @@
             use_v1_sens = not is_test_scan
             if not is_test_scan and not args.v1 and '.sens' in scan_file_types:
                 print('Note: ScanNet v2 uses the same .sens files as ScanNet v1: Press \'n\' to exclude downloading .sens files for each scan')
-                key = raw_input('')
+                key = input('')
                 if key.strip().lower() == 'n':
                     scan_file_types.remove('.sens')
             download_scan(scan_id, out_dir, scan_file_types, use_v1_sens)
@@ -188,10 +185,10 @@
         print('Note that existing scan directories will be skipped. Delete partially downloaded directories to re-download.')
         print('***')
         print('Press any key to continue, or CTRL-C to exit.')
-        key = raw_input('')
+        key = input('')
         if not args.v1 and '.sens' in file_types:
             print('Note: ScanNet v2 uses the same .sens files as ScanNet v1: Press \'n\' to exclude downloading .sens files for each scan')
-            key = raw_input('')
+            key = input('')
             if key.strip().lower() == 'n':
                 file_types.remove('.sens')
         download_release(release_scans, out_dir_scans, file_types, use_v1_sens=True)
